_FIPS201/requirements.md

```diff
@@ -498,7 +498,7 @@ can be reset in certain circumstances.
 
 The PIN on a PIV Card may need to be reset if the cardholder has
 forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an
-invalid PIN more than the allowed number of retries. A maximum of 10 consecutive PIN retries **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
+invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scarring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Comment:
```suggestion
invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has experienced epidermal scarring or similar physical change, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Reply:
@jimfenton agree. Should it be change**s** (plural)? I can't decide. Should we also add text (footnote?) to discourage re-enrollment for temporary damage?

Reply:
Search keyword OT-57 from t[he comment on SP 800-73-4](https://csrc.nist.gov/csrc/media/publications/sp/800-73/4/archive/2015-05-29/documents/sp800_73-4_2013_draft_comments_and_dispositions.pdf).  

There, it is stated that OCC reset is seldom done - making an re-enrollment the most likely outcome instead of a reset.  RESET does not enroll fresh biometric references.  Maybe we need to state that re-enrollment is needed in most cases?